shared/common-adapters/multi-avatar.js

```diff
@@ -22,11 +22,11 @@ const MultiAvatar = ({avatarProps, singleSize, multiSize, style}: Props) => {
     return null
   }
 
-  const leftProps: AvatarProps = avatarProps[0]
-  const rightProps: AvatarProps = avatarProps[1]
+  const leftProps: AvatarProps = avatarProps[1]
+  const rightProps: AvatarProps = avatarProps[0]
```

Comment:
i think the input avatars should be in priority order, so the first should be on top and second behind, but its also rendered right to left so this enforces that. we were actually drawing it incorrectly before. like

avatarProps = [a,b] 
should render
```
[b]
   [a]
```